lib/frame_arch.ml

```diff
@@ -96,4 +98,8 @@ type t =
   | Lm32
   | Microblaze
   | Last
-  [@@deriving enum]
+[@@deriving enumerate]
```

Comment:
Is the type of a module, produced by the `Frame_enum.Make` functor compatible with the old `enum` interface? Do we need to bump the major version in other words?

Reply:
it is. with the only exception that we don't produce functions names in the same way like `enum` does, e,g. if the type name is `insn` then we will not produce function  `insn_of_enum`